examples/benzeneselenol.cjson

```diff
@@ -0,0 +1,279 @@
+{
+  "chemical json": 0,
+  "name": "Benzeneselenol",
+  "inchi": "1S/C6H6Se/c7-6-4-2-1-3-5-6/h1-5,7H",
+  "formula": "C 6 H 6 Se",
+  "atoms": {
+    "elements": {
+      "number": [  6,  6,  6,  6,  6,  6,  1,  1,  1,  1,  1, 34,  1 ]
+    },
+    "coords": {
+      "3d": [  -2.894762	-0.013642	 -0.001528
+               -2.206251	 1.193851	 -0.002521
+               -0.816426	 1.215302	 -0.002201
+               -0.103352	 0.018392	  0.003106
+               -0.790663	-1.194384	  0.00585
+               -2.179957	-1.205971	  0.001789
+               -3.975843	-0.025301  -0.002904
+               -2.750234	 2.129137	 -0.005276
+               -0.296184	 2.163018	 -0.007326
+               -0.247467	-2.130231	  0.013226
+               -2.702896	-2.153075	  0.003664
+                1.821080  -0.043378	 -0.003876
+                2.004358	 1.410007	  0.103449 ]
+    },
+    "partialcharges": {
+      "mulliken": [ -0.49915,  0.056965,  0.172161,  0.349794, -0.153072,  0.094583,
+                     0.016487,  0.050249,  0.002149,  0.01161 ,  0.053777, -0.173671,
+                     0.018118]
+    }
+  },
+  "vibrations": {
+    "frequencies":  [  129.5512   170.6681   231.4278   304.8614   407.8299   472.5026
+                       629.9087   679.9032   693.2509   746.7694   812.5113   850.2578
+                       915.8742   987.1252   988.1785  1002.8922  1038.1073  1091.4005
+                       1102.3417  1183.3857  1209.2727  1311.3497  1355.6441  1471.4447
+                       1510.1919  1611.9088  1619.0156  2391.2487  3165.1596  3171.3909
+                       3182.0753  3188.5786  3198.0359 ]
+    "symmetry": [  u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A',
+                   u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A',
+                   u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A' ]
+    "displacement": [  [  [ 0.    0.   -0.06]
+                          [ 0.    0.   -0.02]
+                          [ 0.    0.    0.03]
+                          ...,
+                          [-0.01  0.   -0.01]
+                          [ 0.    0.   -0.03]
+                          [-0.01 -0.07  0.97]]
+
+                       [  [ 0.    0.01  0.19]
+                          [ 0.    0.    0.03]
+                          [ 0.   -0.01 -0.2 ]
+                          ...,
+                          [-0.02  0.    0.11]
+                          [ 0.    0.    0.04]
+                          [-0.04 -0.04  0.73]]
+
+                       [  [ 0.03 -0.11  0.01]
+                          [-0.13 -0.01  0.  ]
+                          [-0.12  0.2  -0.01]
+                          ...,
+                          [ 0.32 -0.09  0.  ]
+                          [-0.03 -0.08  0.  ]
+                          [ 0.37 -0.14  0.15]]
+
+                          ...,
```

Comment:
The ... is not valid, the arrays have no comma separation. Perhaps this would be better as a document stating proposals rather than an example. Otherwise, we should ensure we have a valid JSON document that can be iterated on.